portfolio/src/main/webapp/index.html

```diff
@@ -34,10 +34,18 @@
         <form action="/data" method="POST">
             <p>Enter a comment</p>
             <textarea name="text-input">Enter a comment here</textarea>
+
+            <label for="quantity">
+                How many comments do you want to view? (between 1 and 5):
+            </label>
+            <input type="number" id="quantity" 
+                   name="quantity" min="1" max="5" />
+            <br />
             <input type="submit" />
         </form>
 
-        <button onClick="getFromServer()">Get from server</button>
+        <button onClick="getFromServer(document.getElementById('quantity').value)">Get from server</button>
```

Comment:
Have `getFromServer` query the quantity value itself. This onclick method should try to be minimal.

Reply:
Will do. Thank you